docs/index.asciidoc

```diff
@@ -494,6 +495,16 @@ A password when using HTTP Basic Authentication to connect to {es}.
 * When present, the event's initial pipeline will _not_ be auto-detected from the event's data stream fields.
 * Value may be a {logstash-ref}/event-dependent-configuration.html#sprintf[sprintf-style] template; if any referenced fields cannot be resolved the event will not be routed to an ingest pipeline.
 
+[id="plugins-{type}s-{plugin}-proxy"]
+===== `proxy`
+
+* Value type is <<uri,uri>>
+* There is no default value for this setting.
+
+Address of the HTTP forward proxy used to connect to the {es} cluster.
+An empty string is treated as if proxy was not set.
+It is useful when using environment variables e.g. `proxy => '${LS_PROXY:}'`.
```

Comment:
Do you mean, it can be used with environment variables?

```suggestion
Environment variables may be used to set this value,  e.g. `proxy => '${LS_PROXY:}'`.
```